packages/core/src/auth/auth.ts

```diff
@@ -728,13 +726,13 @@ export class Auth implements AuthService, ConnectionManager {
     }
 
     /**
-     * Auth processes can fail if there are network issues, and we do not
-     * want to intepret these failures as invalid/expired auth tokens.
+     * Auth processes can fail due to reasons outside of authentication actually being expired.
+     * We do not want to intepret these failures as invalid/expired auth tokens.
      *
-     * We use this to check if the given error is network related and then
-     * throw, expecting the caller to not change the state of the connection.
+     * We use this to check if the given error is unanticipated. If it is we throw,
+     * expecting the caller to not change the state of the connection.
      */
-    private throwOnNetworkError(e: unknown) {
+    private throwOnUnanticipatedError(e: unknown) {
```

Comment:
`throwOnNetworkError` was a good name, since that is the only thing its checking for.

Reply:
Makes sense. Was going to add other types of errors but ended up not.